BoldGrid/weforms
# Fixes contrast for form white text error messages

Fixed contrast in form error messages when text is white or light colored for dark backgrounds. Fixes #177 

### Testing

1. Create a page with a dark background ( Used crio and P&PB )
2. Create a form with at least one required fields
3. Submit the form without filling out the required field.
4. Text should change from white ( Light color ) to black for the error message.
